src/pages/gateway/cicd.md

```diff
@@ -0,0 +1,237 @@
+---
+title: CI/CD for API Mesh
+description: Learn how to set up continuous integration and continuous delivery/deployment for API Mesh .
+keywords:
+  - API Mesh
+  - Extensibility
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# CI/CD for API Mesh
+
+API Mesh for Adobe Developer App Builder now provides CI/CD through [Github actions](https://docs.github.com/en/actions). This allows developers to build, test, and deploy meshes faster and more reliably.
+
+## API Mesh authentication
+
+<InlineAlert variant="info" slots="text"/>
+
+[Create your mesh](./create-mesh.md) before beginning this process, the supplied GitHub action only handles mesh updates.
+
+To get an OAuth token to enable CI/CD:
+
+1. Navigate to the [Adobe Developer Console](https://developer.adobe.com/console) and select your desired workspace.
+
+    ![Workspace Card](../_images/workspace-card.png)
+
+1. Click the **Add Service** button and select API.
+
+1. From the list of APIs, select the **I/O Management API** under the Adobe Services filter and click **Next**.
+
+    ![IO management API](../_images/io-management-api.png)
+
+1. With OAuth Server-to-Server selected, click **Save configured API**.
+
+1. Click **OAuth Server-to-Server** inside the I/O Management API.
+
+1. Click "Generate access token". Copy and save this token for later use.
```

Comment:
The developers need not generate the token. The GitHub workflow will generate the token based on ClientID, Client secret etc.

Reply:
[This](https://developer.adobe.com/developer-console/docs/guides/authentication/ServerToServerAuthentication/implementation/#generate-access-tokens) is a good article to reference, to generate the access token programmatically. Very useful in the case of BYO CI/CD.